### DISCUSSION TITLE
[V8] how can i fetch language file from remote server.

### DISCUSSION THREAD

**Discussion by mehmetpolat** [nuxt-modules/i18n#1825] [General]
how can i fetch language file from remote server.  I tried a lot but I couldn't succeed. Can you share a sample code please?

**Comment by kazupon** [comment]
This is not issue, so we will move to github discussions

**Reply by mehmetpolat** [reply]
Thank you :) I think you can help with this. I really need :)

**Reply by calebeaires** [reply]
@mehmetpolat, have you found a solution?

**Comment by Eddie344** [comment]
@mehmetpolat use this code in lang file
More info: https://v8.i18n.nuxtjs.org/guide/lazy-load-translations
```
export default defineI18nLocale((locale) => {
  // for example, fetch locale messages from nuxt server
  return $fetch(`/api/${locale}`)
})
```
